data_processors/pipeline/lambdas/orchestrator.py

```diff
@@ -71,6 +73,153 @@ def update_step(wfr_id, wfv_id, wfr_event, context):
     return None
 
 
+def get_germline_runs(sequence_run: SequenceRun):
```

Comment:
@reisingerf Let us go through this routine again. On the second look, this may need more a bit more on filtering status. And, also we can enhance/improve into Service layer and WorkflowManager for query, etc. I will explain why so.

Reply:
Yup, that definitely needs some work.
Should take workflow reruns into account as well, I guess.

Reply:
Addressed in 8b7aa66